docs/source/quick_start/first_run.rst

```diff
@@ -0,0 +1,163 @@
+
+First Run
+=========
+
+Now you have installed MCDR, it's time to run it for the first time...?
+
+.. note::
```

Comment:
```rst
.. note::

	The following document uses Minecraft Java Edition server as the demonstrated daemon server,
	as it's the most common use case for MCDR

.. note::

	We assume you have basic knowledge of server management, including:

	* The Minecraft server software: what it is, how to configure it
	* Start command of Minecraft server: what it is, what its arguments mean
	* Console of the server: what it prints, commands you can send to it
	* YAML and JSON files: what they are, how to edit them

    If you have not, Google is your friend
```

Reply:
is this necessary? the first part below is "Minecraft Server Software" with similar statement
maybe just add some wordings such as
```
Most common use of MCDR is to control a Minecraft server. In the following section, we also use this as a demonstration

To do so, you should have a **well-configured** Minecraft server software.
```

Reply:
It's necessary because MCDR can manage servers beyond Minecraft servers, so "knowledge of managing a Minecraft server" is not required to use MCDR. That's why a separate section on "most common use", "use Minecraft server as an example" is necessary

For how this "separate section" is organized, it's on you

Reply:
Please note the changes to the content in https://github.com/MCDReforged/MCDReforged/pull/338#discussion_r1791426897, e.g. `what is it` -> `what it is`